Sources/Commands/SwiftPackageTool.swift

```diff
@@ -259,6 +262,7 @@ public struct SwiftPackageTool: SwiftTool {
         print("OPTIONS:")
         print("  -C, --chdir <path>        Change working directory before any other operation")
         print("  --color <mode>            Specify color mode (auto|always|never)")
+        print("  --enable-code-coverage    Enable code coverage in generatd Xcode projects")
```

Comment:
I don't think we have enough space to go into the semantics of what it means for Xcode, it is too complicated to describe that it actually just means it is enabled by default in the default scheme we generate, I think. What do you think?